src/Core/AdminConsole/Services/Implementations/OrganizationService.cs

```diff
@@ -1040,11 +1038,6 @@ public async Task DisableTwoFactorProviderAsync(Organization organization, TwoFa
             throw new BadRequestException("The Manager role has been deprecated by collection enhancements. Use the collection Can Manage permission instead.");
         }
 
-        if (invites.Any(i => i.invite.AccessAll))
-        {
-            throw new BadRequestException("The AccessAll property has been deprecated by collection enhancements. Assign the user to collections instead.");
-        }
```

Comment:
We no longer need this check because `AccessAll` doesn't exist on the object anymore, so it can't be passed in.